airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.tsx

```diff
@@ -0,0 +1,25 @@
+import classnames from "classnames";
+import React from "react";
+
+import styles from "./NumberBadge.module.scss";
+
+interface NumberBadgeProps {
+  value: number;
+  color?: string;
```

Comment:
This should be typed so we know which value we expect:

```suggestion
  color?: "green" | "red" | "blue" | "default";
```

Also since the lack of the value has currently a darkBlue color, we should make sure there's a way to specify that as well via an explicit value (i.e. `default` in this case), so the logic for the colors below also needs adjustments to handle this.

Reply:
@timroes I'll rename the `darkBlue` to `default` - it's so faded it looks like grey and I think the name is confusing!